packages/components/src/autocomplete/index.tsx

```diff
@@ -34,100 +36,42 @@ import { speak } from '@wordpress/a11y';
  */
 import { getAutoCompleterUI } from './autocompleter-ui';
 import { escapeRegExp } from '../utils/strings';
-
-/**
- * A raw completer option.
- *
- * @typedef {*} CompleterOption
- */
-
-/**
- * @callback FnGetOptions
- *
- * @return {(CompleterOption[]|Promise.<CompleterOption[]>)} The completer options or a promise for them.
```

Comment:
Interestingly, in the original types `options` was either an array, or a promise returning an array. No signs of it being a function returning an array?

Reply:
You make a good point, there is a disconnect between the original types and the README description.
Original types list an array of options or a promise returning an array.
[README lists](https://github.com/WordPress/gutenberg/blob/086b77ed409a70a6c6a6e74dee704851eff812f2/packages/components/src/autocomplete/README.md#L85-L86) an array, a function that returns an array, or a function that returns a promise for an array

Reply:
marking this conversation as resolved, since the same topic is being discussed in https://github.com/WordPress/gutenberg/pull/47751#discussion_r1107705071